src/signals-unix.c

```diff
@@ -84,12 +84,24 @@ static void jl_call_in_ctx(jl_ptls_t ptls, void (*fptr)(void), void *_ctx)
     *(void**)rsp = NULL;
     ctx->uc_mcontext.gregs[REG_RSP] = rsp;
     ctx->uc_mcontext.gregs[REG_RIP] = (uintptr_t)fptr;
+#elif defined(_OS_FREEBSD_) && defined(_CPU_X86_64_)
```

Comment:
This change and the other 2 sections in `jl_call_in_ctx` do not backport cleanly to release-0.5 since #17727 was not backported. Okay to just not change `jl_call_in_ctx` at all there?